snowflake/README.md

```diff
@@ -74,7 +74,7 @@ datadog-agent integration install datadog-snowflake==2.0.1
         ## For more information on Snowflake account names,
         ## see https://docs.snowflake.com/en/user-guide/connecting.html#your-snowflake-account-name
         #
-      - account: <ACCOUNT>
+      - account: <ORG_NAME>-<ACCOUNT_NAME>
```

Comment:
Could you also update it here in `spec.yaml`, and then sync this to the sample `conf.yaml` with `ddev validate config snowflake -s`?